test/unit/rate-limit-test.spec.js

```diff
@@ -114,154 +119,156 @@ test('Retry on 5** - multiple errors - reach/exceed limit', (t) => {
   mock.onGet('/rate-limit-me').replyOnce(200, 'works')
   mock.onGet('/rate-limit-me').replyOnce(500, 'Server Error', { 'x-contentful-request-id': 12345 })
 
-  t.plan(3)
+  // expect.assertions(3)
 
-  return client
-    .get('/rate-limit-me')
-    .then((response) => {
-      t.ok(response.data)
-      t.equals(response.data, 'works')
-    })
-    .then(() => {
-      return client
-        .get('/rate-limit-me')
-        .then(() => {
-          t.fail('Promise should reject not resolve')
-          teardown()
-        })
-        .catch((error) => {
-          t.equals(error.message, 'Request failed with status code 500')
-          teardown()
-        })
-    })
+  return client.get('/rate-limit-me').then((response) => {
+    expect(response.data).toBeDefined()
+    expect(response.data).toEqual('works')
+    done()
+  })
+  // .then(() => {
```

Comment:
What's with these tests?

Reply:
Actually, everything is fine I just forgot to uncomment this part 🤕